src/win/tty.c

```diff
@@ -1004,6 +1004,7 @@ int uv_tty_read_start(uv_tty_t* handle, uv_alloc_cb alloc_cb,
   if (handle->tty.rd.last_key_len > 0) {
     SET_REQ_SUCCESS(&handle->read_req);
     uv_insert_pending_req(handle->loop, (uv_req_t*) &handle->read_req);
+    handle->reqs_pending++;
```

Comment:
can you move this after setting the flag? that's how it's done in other places in the code. (consistency yada yada yada .... )